doc/man1/flux-restore.rst

```diff
@@ -0,0 +1,82 @@
+.. flux-help-description: Restore kvs from portable archive format
+
+===============
+flux-restore(1)
+===============
+
+
+SYNOPSIS
+========
+
+**flux** **restore** [*OPTIONS*] *INFILE*
+
+
+DESCRIPTION
+===========
+
+The ``flux-restore`` command reads a KVS snapshot from a portable archive
+format, usually written by :man1:`flux-dump`.
+
+The archive source may be specified as a file path or *-* for standard input.
+The format of the archive may be any of the formats supported by
+:linux:man3:`libarchive` and is determined on the fly based on the archive
+content.
+
+The snapshot may be restored to a KVS key if *--key=NAME* is used and the
+KVS service is running, or as a checkpoint in the content backing store
+if *--checkpoint* is used, without the KVS running.  One of those two options
+is required.
+
+
```

Comment:
perhaps want to add (or add an example?) of how dump + restore garbage collects inactive references?  But on other hand it may not be super wise to advertise, as users may abuse it?

Reply:
I think it may be best to hold off on that for a bit.  For now, the things that are omitted from the archive are described in `flux-dump(1)`.  If we're successful with using it for garbage collection, we could revisit.